release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+
+{product-title-short} includes features that depend on the new architecture. Specifically, policy criteria and vulnerability reporting with collections were introduced in 3.74 as a Technology Preview, and are now generally available.
+
+//update link to Customer Portal link after CP version is published (lags behind by ~1 day)
+For instructions for a new installation of {product-title-short} 4.0, see xref:../installing/acs-installation-platforms.adoc[Supported platforms and installation methods].
+
+[id="upgrade-to-40"]
+=== Upgrading {product-title-short} to version 4.0
+
+To prevent automatic upgrades when using the Operator, {product-title-short} 4.0 uses a new subscription channel. This update allows the {product-title-short} Operator to comply with Red Hat standards and provides consistency with other Red Hat Operators. As documented in the upgrade instructions, you must explicitly change your subscription channel when upgrading. Customers who remain on the `latest` channel in the current {product-title-short} Operator channel will continue to receive 3.74 updates until it is no longer supported. The life cycle of {product-title-short} 3.74 has been extended by 3 months to allow you more time to migrate to {product-title-short} 4.0. The upgrade process includes automatically migrating the database, and requires no intervention.
+
+After Central is upgraded, upgrade the subscription channel on all Secured Clusters.
+
+You are encouraged to evaluate the upgrade in a staging environment before pushing it to production to ensure that your unique environment does not present unforeseen issues.
+
+The documentation provides detailed instructions for rolling back to the previous version if necessary. Before the upgrade, back up the existing Central database following the xref:../backup_and_restore/backing-up-acs.adoc#backing-up-acs[documented backup procedure] so that you can roll back to the previous version if necessary. You are encouraged to practice rolling back to the previous version in the staging environment to ensure that your backup was successful and that rolling back the previous version brings the system back to the expected operational state.
+
+To upgrade to 4.0, {product-title-short} perform the following steps:
+
+. Upgrade to {product-title-short} version 3.74 if you have an earlier version installed.
+. Back up the database for Central.
+. Upgrade Central to version 4.0.
+. Upgrade secured clusters to version 4.0.
+
+For additional instructions on upgrading, see the documentation in the "Upgrade documentation" section. Procedures for upgrading by using Helm and `roxctl` now include a step to back up the Central database before upgrading Central.
+
+[id="upgrade-acscs-rn40"]
+=== Upgrading {rh-rhacscs-first} to version 4.0
+
+To upgrade, {product-title-managed-short} you must switch the Operator channel to `stable` for your secured clusters.
```

Comment:
[Sorry, missed it on the first pass] Here we can also say that `roxctl` (manifest-based) and Helm upgrade procedures for Secured Clusters remain unchanged (from before 4.0).